CHANGES.rst

```diff
@@ -1,17 +1,21 @@
 ``zope.event`` Changelog
 ========================
 
-4.1.1 (unreleased)
+4.2.0 (unreleased)
 ------------------
 
-- Claim support for Python 3.5.
+- Add support for Python 3.5.
```

Comment:
"State" or "note," maybe? I like the distinction, subtle as it may be, although the fact that this is the first time I've heard it put that way may speak to how widespread it is or isn't.